test/services/mapping/declaration/DeclarationBuilderSpec.scala

```diff
@@ -75,7 +75,7 @@ class DeclarationBuilderSpec extends WordSpec with Matchers {
 
       declaration.getTypeCode.getValue should be("EXY")
 
-      assertGoodsShipment(declaration)
+//      assertGoodsShipment(declaration)
```

Comment:
If we try deleting the DeclarationBuilder we should be able to get rid of this class